dev-client/src/screens/LocationScreens/components/TopSoilMatchesInfoContent.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useTranslation} from 'react-i18next';
+import {Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+export const TopSoilMatchesInfoContent_Site = () => {
```

Comment:
```suggestion
export const TopSoilMatchesInfoSiteDescription = () => {
```
question: will this have more than the description? that will help name it. If we aren't showing anything else, then perhaps a single component with a siteId or isSite prop would be simpler.

Suggest avoiding underscores for internal consistency.